.gitignore

```diff
@@ -38,7 +38,14 @@ _ocamltestd
 .merlin
 _build
 META
+
+# VScode, Emacs, VIM temporary files
+
 .vscode
+*~
+#*#
```

Comment:
I think these hashes need to be backslash-escaped.

(No strong opinion either way on the change itself. I agree we should include these if we're keeping #12504)

Reply:
Well spotted, thanks!  Fixed in 181587a.